### DISCUSSION TITLE
"TypeError: Cannot read properties of undefined (reading 'Root')" while using ScrollArea Component

### DISCUSSION THREAD

**Discussion by Keshav7802** [huntabyte/shadcn-svelte#882] [Help]
Hello everyone,

I'm trying to use the ScrollArea Component from shadcn-svelte, but I'm facing TypeError as stated in title. I installed the scrollArea, and tried to run the first example on this page [https://www.shadcn-svelte.com/docs/components/scroll-area](url). Anyone who knows please help me.

**Comment by shyakadavis** [comment]
Hi, @Keshav7802 

May you share a snippet?

But it sounds like an issue with import statements, or updating your `bits-ui` version.

Cross-check both how you're using it and the `bits-ui` version.

Good luck.

**Reply by Keshav7802** [reply]
![image](https://github.com/huntabyte/shadcn-svelte/assets/65281740/17cadfbe-d4d8-4f6c-9127-ca6848e9c9a6)

After this I tried to route this code to a link

![image](https://github.com/huntabyte/shadcn-svelte/assets/65281740/c4377c9f-ad8e-41d5-af2e-5e3298d1915f)

And I'm getting this as output

![image](https://github.com/huntabyte/shadcn-svelte/assets/65281740/4bac47e2-b116-4fcd-9fad-facc79a40196)

and this error

![image](https://github.com/huntabyte/shadcn-svelte/assets/65281740/d327e45a-1327-432a-84da-d727cae205f2)

**Reply by shyakadavis** [reply]
Could you try

```
npm i bits-ui@latest
```

I can't see your version from the pictures :(

If it still fails, could try a repro from here: https://www.shadcn-svelte.com/repro

**Reply by Keshav7802** [reply]
Thanks it worked.
